src/imp/openssl.rs

```diff
@@ -303,6 +303,18 @@ impl TlsConnector {
     }
 }
 
+impl fmt::Debug for TlsConnector {
```

Comment:
I would just derive this, even if it includes the context field. If that gets a different implementation upstream then it'll get picked up here.

Reply:
To clarify Debug implemented on SslContext but not SslConnector which is why I didn’t derive it here.

I’m happy to make the change in rust-openssl if you can help shepherd it through since we’d need to land that before this one. :)

Reply:
Oh, got it. The manual implementation makes sense then. We should implement Debug for SslConnector at some point for sure though.

Reply:
Done! https://github.com/sfackler/rust-openssl/pull/1288